app/models/user_emails.py

```diff
@@ -0,0 +1,23 @@
+from app.models import db
+
+
+class UserEmails(db.Model):
```

Comment:
This will also require a boolean field for verified emails.

Reply:
done